odoo/addons/test_apikeys/static/tests/apikey_flow.js

```diff
@@ -56,7 +56,7 @@ registry.category("web_tour.tours").add('apikeys_tour_setup', {
         $('button:contains("Done")').click();
     }
 }, {
-    content: "check that our key is present",
+    content: "check that our key is present (FIXME: requires HR to be installed)",
```

Comment:
`hr` changes the top-right profile preferences from a modal to a form view. When it is a modal this step breaks because the modal is automatically closed by the previous step. When it is a form view, the form view stays open hence this step passes. Ideally we shouldn't depend on HR but I'm kinda lazy to fix this tour atm.